paths/paths.go

```diff
@@ -62,16 +63,17 @@ func Exercises(path string) string {
 	return expandPath(path)
 }
 
-// Recalculate sets exercism paths based on Home.
-func Recalculate() {
-	if Home == "" {
-		home, err := findHome()
-		if err != nil {
-			panic(err)
-		}
+// Recalculate sets exercism paths
+func Recalculate(home, configHome string) {
```

Comment:
It seems this functions whole purpose is to setup the `Home` and `ConfigHome` and only used within `init`. What about just dropping the function all together and setting the variables directly on the tests and doing this same logic from init?